examples/js/loaders/GLTFLoader.js

```diff
@@ -2160,6 +2270,18 @@ THREE.GLTFLoader = ( function () {
 
 			}
 
+			materialType = this._invokeOne( function ( ext ) {
+
+				return ext.getMaterialType && ext.getMaterialType();
```

Comment:
`getMaterialType()` should take an argument `materialDef` so that you can optionally use it to determine if a material applies to just this material definition.

Reply:
Agreed – I'd vote for `materialIndex` for consistency with load methods.

Reply:
Sounds good to me. Added `materialIndex`, thanks.